<ui version="4.0" >
 <class>SyrthesForm</class>
 <widget class="QWidget" name="SyrthesForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>603</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>398</width>
     <height>446</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="labelSyrthes" >
          <property name="toolTip" >
           <string>Preprocessor option: -syrthes</string>
          </property>
          <property name="text" >
           <string>Syrthes coupling:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" >
         <widget class="QRadioButton" name="radioButtonCouplingOn" >
          <property name="text" >
           <string>on</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QRadioButton" name="radioButtonCouplingOff" >
          <property name="text" >
           <string>off</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="groupBox" >
        <property name="statusTip" >
         <string>Boundary faces selections for Syrthes coupling. These selections are optional. A selection can be activated or not. </string>
        </property>
        <property name="title" >
         <string/>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="labelFaces" >
             <property name="toolTip" >
              <string/>
             </property>
             <property name="text" >
              <string>Select boundary faces for Syrthes coupling:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="FacesSelectionView" native="1" name="widgetFaces" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>350</width>
               <height>250</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="Line" name="line" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label" >
               <property name="text" >
                <string>Project mesh for 2D syrthes run:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QRadioButton" name="radioButton2DOn" >
               <property name="text" >
                <string>on</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" >
              <widget class="QRadioButton" name="radioButton2DOff" >
               <property name="text" >
                <string>off</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FacesSelectionView</class>
   <extends>QWidget</extends>
   <header>FacesSelectionView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonCouplingOff</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>311</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>354</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCouplingOn</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>264</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>235</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
